Remotion/ObjectBinding/Web/UI/Controls/Validation/BocListValidationResultDispatchingValidator.cs

```diff
@@ -130,12 +131,14 @@ private void DispatchToValidationFailureRepository (IBusinessObjectValidationRes
 
     protected override bool EvaluateIsValid ()
     {
+      //Do we want to reduce this, as we do lazy validation anyways? We might not want to have the error message being built in here as it's never shown anyways
```

Comment:
Yeah maybe it is best to use just a generic message here and not handle any of the validation failures in the repo. This way, we have the final evaluation in the lazy method and can ensure that we get all the remaining unhandled validation failures. This is important as any validation failure that is left in the repo after this point is just gone. So we should ensure that we print all the relevant error messages there.